{{cookiecutter.project_slug}}/Dockerfile

```diff
@@ -1,25 +1,33 @@
-FROM ubuntu:bionic
+FROM python:3.7-slim-buster as base
 
-ENV LANG C.UTF-8
-ARG DEBIAN_FRONTEND=noninteractive
-# Allow SECRET_KEY to be passed via arg so collectstatic can run during build time
-ARG SECRET_KEY
-
-# libpq-dev and python3-dev help with psycopg2
 RUN apt-get update \
-  && apt-get install -y python3.7-dev python3-pip libpq-dev curl \
-  && apt-get clean all \
+  && apt-get install -y python3.7-dev python3-pip libpq-dev \
   && rm -rf /var/lib/apt/lists/*
   # You can add additional steps to the build by appending commands down here using the
   # format `&& <command>`. Remember to add a `\` at the end of LOC 12.
   # WARNING: Changes to this file may cause unexpected behaviors when building the app.
   # Change it at your own risk.
 
 WORKDIR /opt/webapp
+
+COPY Pipfil* ./
+RUN pip3 install --no-cache-dir cookiecutter -q 'pipenv==2018.11.26' \
+    && pipenv install --deploy --system
+
 COPY . .
-RUN pip3 install --no-cache-dir -q 'pipenv==2018.11.26' && pipenv install --deploy --system
+ARG SECRET_KEY
 RUN python3 manage.py collectstatic --no-input
 
+FROM python:3.7-slim-buster
```

Comment:
base image can be re-used here.

`FROM base as release`

Reply:
Noted.

Reply:
fixed